src/systems/settling.cairo

```diff
@@ -0,0 +1,289 @@
+#[system]
+mod Settle {
+    use traits::Into;
+    use box::BoxTrait;
+
+    use eternum::utils::unpack::unpack_resource_ids;
+    use eternum::constants::WORLD_CONFIG_ID;
+    use eternum::interfaces::{IERC721Dispatcher, IERC721DispatcherTrait};
+    use eternum::erc721::erc721::{RealmData, Position};
+    use eternum::components::owner::Owner;
+    use eternum::components::realm::Realm;
+    use eternum::components::resources::Resource;
+    use eternum::components::age::Age;
+    use eternum::components::config::WorldConfig;
+
+    fn execute(realm_id: felt252) { // get the ERC721 contract
+        // get the owner
+        let config = commands::<WorldConfig>::entity(WORLD_CONFIG_ID.into());
+        let token = config.realm_l2_contract;
+        let owner = commands::<Owner>::entity((token, realm_id).into());
```

Comment:
Why are we getting Owner here? Wouldn't the address be 0 on settling? Besides that, why do we even need it if the Realm component has a `owner` member? Or maybe it's the the other way around, should the Realm component have `owner` at all? Wouldn't it be better to store it in a separate component, the `Owner` here?

Reply:
agreed - If we are storing Owner in a separate component, then it's not needed on the Realm component

Reply:
agreed, let's remove the owner field and only keep `Owner` component !